profile/forms.rkt

```diff
@@ -607,6 +607,16 @@ for the forms are run.
                    check-values
                    100000)))
 
+(module partition "forms-base.rkt"
+  (provide run)
+  (define (parition . vs)
```

Comment:
Btw, I've been bitten before by this step in the CI workflow reporting as succeeded even though it had an error. It's because the `|` used in this command obscures the error result of the first command in the pipeline. Should be able to fix this in the future: https://unix.stackexchange.com/questions/14270/get-exit-status-of-process-thats-piped-to-another